exercises/practice/propositional-logic/.meta/config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "blurb": "A logic exercise. Implement a function that returns three common propositional logic proofs",
+  "authors": [
+    "JimmyQuenichet"
+  ],
+  "contributors": [
+    "CSwift8",
+    "Newby555",
+    "andjam19",
+    "grantg012"
```

Comment:
You don't need to put anyone as a contributor unless they actually worked on the code or did the config file